<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="3" total="4" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-15T17:32:26 IST" name="My Store" finished-at="2021-12-15T17:35:12 IST" duration-ms="166708">
    <groups>
    </groups>
    <test started-at="2021-12-15T17:32:26 IST" name="My Store" finished-at="2021-12-15T17:35:12 IST" duration-ms="166708">
      <class name="Pages.My_Store_Login">
        <test-method signature="Login()[pri:0, instance:Pages.My_Store_Login@8f57e4c]" started-at="2021-12-15T17:32:26 IST" name="Login" finished-at="2021-12-15T17:35:12 IST" duration-ms="166565" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[verificaiton of result expected [verificaiton of result] but found [Product successfully added to your shopping cart]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: verificaiton of result expected [verificaiton of result] but found [Product successfully added to your shopping cart]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at Pages.My_Store_Login.Login(My_Store_Login.java:53)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:599)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Login -->
      </class> <!-- Pages.My_Store_Login -->
      <class name="Base.My_Store_Base">
        <test-method is-config="true" signature="initialize()[pri:0, instance:Base.My_Store_Base@722531ab]" started-at="2021-12-15T17:32:03 IST" name="initialize" finished-at="2021-12-15T17:32:24 IST" duration-ms="21007" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialize -->
        <test-method is-config="true" signature="setup()[pri:0, instance:Base.My_Store_Base@722531ab]" started-at="2021-12-15T17:32:24 IST" name="setup" finished-at="2021-12-15T17:32:26 IST" duration-ms="1430" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:Base.My_Store_Base@722531ab]" started-at="2021-12-15T17:35:12 IST" name="tearDown" finished-at="2021-12-15T17:35:15 IST" duration-ms="2303" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- Base.My_Store_Base -->
    </test> <!-- My Store -->
  </suite> <!-- My Store -->
</testng-results>
